app/controllers/api/base.rb

```diff
@@ -1,5 +1,9 @@
 module API
   class Base < Grape::API
+    rescue_from ReadOnlyAccessControl::DisallowedUpdate do
+      rack_response({ developer_message: "The i14y API is currently in read-only mode.", status: 503 }.to_json, 503)
```

Comment:
Not a nit, but down the line we might consider adding details to the response including the maintentance window start/end times.